### DISCUSSION TITLE
Add popup to unclustered point inside a cluster

### DISCUSSION THREAD

**Discussion by gurdenbatra** [uber/react-map-gl#1601] [Q&A]
Hey, I used the [cluster example ](https://visgl.github.io/react-map-gl/examples/clusters), when I zoom into the cluster can see the unclustered points, I want to able to show up popup with data from the geoJSON file. Like in the example given on [mapbox-gl](https://docs.mapbox.com/mapbox-gl-js/example/cluster/) site. 

I tried few things but still not able to add a popup to the unclustered points/markers, any idea how I should proceed? Thanks a lot for the great library.

**Comment by sheethalpatila** [comment]
you got any solutions ?
